### PR TITLE
Embedding Scratch 3 projects w/ menu toggle.

### PR DESCRIPTION
**Pull request by infchem**

### CODE REVIEW

File: EmbedScratch.php [hunk 1]
```diff
@@ -58,8 +59,15 @@ function renderEmbedScratch ($input, $argv, $parser) {
 		{
 			$height = $argv['height'];
 		}
+		if (
+			!empty( $argv['menu'] ) &&
+			settype( $argv['menu'], 'integer' )
```
Comment (Kenny2github) [inline EmbedScratch.php:64]: What's the purpose of this?

Comment (infchem) [reply EmbedScratch.php:64]: Hiding the blue menu (margin-top:-52px) or not.

Comment (Kenny2github) [reply EmbedScratch.php:64]: This blue menu?
image
I don't think that should ever appear... why is an option necessary?

**Comment by Kenny2github** [comment]
By the way, for future reference you don't have to close and re-open pull requests for new commits - if you make another commit to the same branch and push then it will be added to the pull request.

File: EmbedScratch.php (hunk 1 above)
Comment (infchem) [reply EmbedScratch.php:64]: No, not this menu :)
The one with the green flag. If you've a project that doesn't rely on the flag, you should be able to hide it. See both screenshots below.
menu
nomenu

Comment (infchem) [reply EmbedScratch.php:64]: It's more greyish, not blue. Sorry for that confusion...

Comment (Kenny2github) [reply EmbedScratch.php:64]: Ah, I see. I suppose that's good to have.

**Comment by apple502j** [comment]
I disagree with "menu hiding" - we should be able to stop projects from running.

**Pull request closed without merging**